daffodil-codegen-c/src/main/scala/org/apache/daffodil/codegen/c/DaffodilCExamplesGenerator.scala

```diff
@@ -81,19 +81,23 @@ object DaffodilCExamplesGenerator {
     val nestedRootName = Some("NestedUnion")
     val padTestSchema = schemaDir / "padtest.dfdl.xsd"
     val padTestRootName = None
+    val simpleSchema = schemaDir / "simple.dfdl.xsd"
+    val simpleRootName = Some("simple-byte")
     val variableLenSchema = schemaDir / "variablelen.dfdl.xsd"
     val variableLenRootName = Some("expressionElement")
 
     val examplesDir = os.Path(args(0))
     val exNumsExampleDir = examplesDir / "ex_nums"
     val nestedExampleDir = examplesDir / "NestedUnion"
     val padTestExampleDir = examplesDir / "padtest"
+    val simpleExampleDir = examplesDir / "simple"
     val variableLenExampleDir = examplesDir / "variablelen"
```

Comment:
Doesn't have to be done in this PR, but at some point we may want to turn this into an array of tuples and we can just iterate over them, maybe something like:
```scala
val examples = Array(
  ("ex_nums.dfdl.xsd", None, "ex_nums"),
  ("nested.dfdl.xsd", Some("NestedUnion"), "NestedUnion")
  ...,
)
examples.foreach { case (schema, rootOpt, dir) =>
  updateCExample(schemaDir / schema, rootOpt, examplesDir / dir)
}
```
Then adding new examples becomes just adding a new entry in the array and things aren't spread out among different variables.

We're starting to get enough examples where this could clean things up a bit.

Reply:
Good idea.  I've replaced the multiple schema, root, dir variables with the Array of tuples.